doc/Doxyfile

```diff
@@ -1089,7 +1089,7 @@ FILTER_SOURCE_PATTERNS =
 # (index.html). This can be useful if you have a project on for instance GitHub
 # and want to reuse the introduction page also for the doxygen output.
 
-USE_MDFILE_AS_MAINPAGE = main.md
+USE_MDFILE_AS_MAINPAGE = _doxygen/main.md
```

Comment:
yeah, Carles has done a fix in the Zephyr side - https://github.com/zephyrproject-rtos/example-application/pull/90. He will update our version as well.